maven-scm-providers/maven-scm-provider-hg/src/main/java/org/apache/maven/scm/provider/hg/command/add/HgAddConsumer.java

```diff
@@ -54,17 +52,17 @@ public void doConsume( ScmFileStatus status, String trimmedLine )
             File tmpFile = new File( workingDir, trimmedLine );
             if ( !tmpFile.exists() )
             {
-                if ( getLogger().isWarnEnabled() )
+                if ( logger.isWarnEnabled() )
                 {
-                    getLogger().warn( "Not a file: " + tmpFile + ". Ignored" );
+                    logger.warn( "Not a file: " + tmpFile + ". Ignored" );
```

Comment:
why not changing/simplify this as well by removing the if and `logger.warn( "Not a file: {}. Ignored", tmpFile );`
this apply to plenty of lines

Reply:
This PR is a mechanical replacement. Placeholders are next, but are right of course.